utils/test_results.py

```diff
@@ -76,131 +298,152 @@ def generate_test_results(
     :param branch: optional name of the branch we want to filter on, if this is provided the aggregates calculated will only take into account
         test instances generated on that branch. By default branches will not be filtered and test instances on all branches wil be taken into
         account.
-    :param history: timedelta for filtering test instances used to calculated the aggregates by time, the test instances used will be
-        those with a created at larger than now - history.
+    :param interval: timedelta for filtering test instances used to calculated the aggregates by time, the test instances used will be
+        those with a created at larger than now - interval.
     :param testsuites: optional list of testsuite names to filter by
```

Comment:
test suites filtering is also union right?

Reply:
indeed i will add that as a comment